scripts/rollup/rollup.config.js

```diff
@@ -179,7 +180,18 @@ module.exports = {
     plugins: [
         nodeResolve({
             // These are the devDeps that may be inlined into the dist/ bundles
-            resolveOnly: [/^@lwc\//, 'observable-membrane'],
+            // These include packages owned by us (LWC, observable-membrane), as well as parse5,
+            // which is bundled because it makes it simpler to distribute
+            resolveOnly: [
+                /^@lwc\//,
+                'observable-membrane',
+                // Special case - parse5 is bundled only in @lwc/engine-server currently
+                packageName === '@lwc/engine-server' && 'parse5',
```

Comment:
We can probably remove this special case when we upgrade `parse5`, and bundle it into `@lwc/template-compiler` as well. For right now I wanted to avoid boiling the ocean.

Reply:
Is that something worth adding a comment for?